src/handlers/store-transactions/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface CleanedEventBody {
```

Comment:
I could imagine putting the type-guard for this close to the type definition so that you always think of updating both.

Reply:
yeah that sounds sensible, I've moved the guard into this file